### DISCUSSION TITLE
anyone writing a Typescript port?

### DISCUSSION THREAD

**Discussion by mindplay-dk** [microsoft/autogen#100] [Ideas]
This project and the results look really incredible!

But coding blind in python, with no typechecks or autocompletion, ugh... is anyone else thinking about porting this to Typescript?

**Comment by sonichi** [comment]
This is a common question I'm keeping track of. I created an issue: #236 
Please feel free to make suggestions. 
BTW I don't code TypeScript. So it'll be nice to have an expert to take the lead of it.

**Reply by mindplay-dk** [reply]
Well, I don't really code in Python, so... 🙂

**Reply by sonichi** [reply]
So, let's find someone who know both? BTW, are you on Discord?

**Comment by chadly** [comment]
anyone tried asking GPT-4 to do it? 😁
